nova-app/src/app/styles/theme/light.css

```
@@ -47,6 +47,7 @@
   --stroke-data-selected: var(--color-light-full);
 
   /* ---------- Shadow ---------- */
+  --shadow-modal: 1px -1px 10px rgba(0, 0, 0, 0.25), 0px 4px 4px rgba(0, 0, 0, 0.25);
```

Stylelint (17.2.0)

[error] Expected "0.25" to be "25%" (alpha-value-notation)

(alpha-value-notation)